homeassistant/components/proxmoxve/binary_sensor.py

```diff
@@ -105,3 +108,9 @@ def available(self):
         """Return sensor availability."""
 
         return super().available and self.coordinator.data is not None
+
+    @property
+    def device_class(self):
+        """Return device class."""
+
+        return BinarySensorDeviceClass.RUNNING
```

Comment:
This is a static value, could you use a class attribute instead?

```python
class ProxmoxBinarySensor(ProxmoxEntity, BinarySensorEntity):
    """A binary sensor for reading Proxmox VE data."""

    _attr_device_class = BinarySensorDeviceClass.RUNNING

    def __init__(
```

Reply:
Great idea! Didn't realize that was possible. I will make the changes now